common/gtk-3.0/meson.build

```diff
@@ -35,46 +35,39 @@ endif
 
 # render PNG assets
 
-gtk3_asset_names = run_command(
+gtk3_assets_names = run_command(
   'cat', gtk3_ver / 'assets.txt',
   check : true
 ).stdout().split()
 
-assets_svg = gtk3_ver / 'assets.svg'
+gtk3_assets_svg = gtk3_ver / 'assets.svg'
 
-gtk3_assets = []
-gtk3_hidpi_assets = []
-
-foreach asset : gtk3_asset_names
-  gtk3_assets += custom_target(
-    'gtk3-' + asset,
-    input : assets_svg,
-    output : asset + '.png',
-    command : [
-      inkscape,
-      '--export-id-only',
-      inkscape_ver.version_compare('>=1.0') ? '--export-filename=@OUTPUT@' : '--export-png=@OUTPUT@',
-      '--export-id=' + asset,
-      '--export-dpi=96',
-      '@INPUT@'
-    ],
-    build_by_default : true
-  )
+gtk3_assets_dpis = []
+gtk3_assets_dpis += {'suffix': '', 'value': '96'}
+gtk3_assets_dpis += {'suffix': '@2', 'value': '192'}
```

Comment:
I used literal strings as key to stay compatible with meson < 0.53.

DPIs are added one per line, to allow to activate/deactivate them conditionally if needed in the future.